.github/workflows/integration_test.yml

```diff
@@ -112,7 +112,7 @@ jobs:
           helm repo add localstack-repo http://helm.localstack.cloud
           helm upgrade --install localstack localstack-repo/localstack --version 0.6.14
           pip install awscli
-          kubectl wait --timeout=60s --for=condition=ready --all pod
+          kubectl wait --timeout=160s --for=condition=ready --all pod
```

Comment:
I remembered we've already fixed the integration test, so this change could be removed.